certbot/storage.py

```diff
@@ -520,6 +520,22 @@ def next_free_version(self):
         # for the others.
         return max(self.newest_available_version(x) for x in ALL_FOUR) + 1
 
+    def ensure_deployed(self):
+        """Make sure we've deployed the latest version.
+
+        :returns: False if a change was needed, True otherwise
+        :rtype: bool
+
+        May need to recover from rare interrupted / crashed states."""
+
+        if self.has_pending_deployment():
+            logger.warn("Found a new cert /archive/ that was not linked to in /live/; "
+                        "fixing and reinstalling..")
```

Comment:
"reinstalling" is only true in the `run` case. When using `renew` or `certonly`, the cert is not reinstalled. For this reason, I think we should remove this word. If we really want, we can add logic to the `run` function to print a message about reinstalling the existing certificate if the action is "reinstall".